tests/io/test_OutputFileFactory.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
+from contextlib import contextmanager
+from io import StringIO
+from pathlib import Path
+
+import netCDF4
+import numpy
+import pytest
+
+from dfastmi.io.FouFile import FouFile
+from dfastmi.io.MapFile import MapFile
+from dfastmi.io.OutputFileFactory import OutputFileFactory
+from tests.io.test_OutputFile import TestOutputFile
+
+
+class Test_OutputFileFactory:
+
+    def test_creation_of_MapFileType(self):
+        """
+        Testing if generate Mapfile is donw with key
+        """
+        output_file = OutputFileFactory.generate("myfile_net.nc")
```

Comment:
should be like this right?

```suggestion
        output_file = OutputFileFactory.generate("myfile_map.nc")
```